libs/npress/AdminModule/presenters/PagesPresenter.php

```diff
@@ -241,6 +261,34 @@ public function handleDeletePage($undo = false)
     }
   }
 
+  public function actionHistory($id_page)
+  {
+    $historyPage = PagesModel::getAllExceptLast($id_page);
+    $this->template->historyPage = $historyPage;
+    $last_version = PagesModel::lastVersion($id_page);
+    $this->template->last_version = $last_version;
+    $this->template->current = $id_page;
+  }
+
+  public function handleback($id_page)
+  {
+    $prevVer = PagesModel::prevVer($id_page);
+    PagesModel::rollBackPage($id_page, $prevVer->lang, $prevVer->name, $prevVer->seoname, $prevVer->heading, $prevVer->text);
+    PagesModel::addPagesHistory($prevVer->id_page, $prevVer->lang, $prevVer->name, $prevVer->seoname, $prevVer->heading, $prevVer->text);
```

Comment:
1.Myslím že tlačítko vzít zpět by mělo tu stránku dostat do přesně stejného stavu jako byl předtím.

Tj. uložit do `pages` $prevVer a smazat poslední řádek v `pages_history`

2. `rollbackPage` myslím duplikuje chování tohoto `PagesModel::getPageById($id_page)->save($prevVer)`. Chtělo by to vyzkoušet a `rollbackPage` kdyžtak smazat.